posthog-react-native/src/surveys/components/SurveyModal.tsx

```diff
@@ -22,13 +22,14 @@ export function SurveyModal(props: SlypModalProps): JSX.Element | null {
   const onClose = useCallback(() => props.onClose(isSurveySent), [isSurveySent, props])
   const insets = useOptionalSafeAreaInsets()
 
-  const surveyPopupDelayMilliseconds = appearance.surveyPopupDelaySeconds * 1000
-  const [isVisible, setIsVisible] = useState(surveyPopupDelayMilliseconds === 0)
-  if (surveyPopupDelayMilliseconds > 0) {
-    setTimeout(() => {
-      setIsVisible(true)
-    }, surveyPopupDelayMilliseconds)
-  }
+  // TODO: delay seconds
```

Comment:
commenting out more possible source of issues so we can focus on the MVP and basic features.